packages/components/src/templates/next/components/internal/Footer/ClientCopyrightYear.tsx

```diff
@@ -0,0 +1,5 @@
+"use client"
+
+export const ClientCopyrightYear = () => {
+  return <>&copy; {new Date().getFullYear()} </>
```

Comment:
nit: I think the space should be part of Footer.tsx rather than here? At first glance it wasn't clear why there was an additional space here.

Reply:
thanks for catching this! changed it in https://github.com/opengovsg/isomer/pull/970/commits/077edc12143eaf837f7a1ed5357659317d281b44